src/js/index.js

```diff
@@ -1,48 +1,90 @@
 import "../styles/main.scss";
 
-const $ = (selector) => document.querySelector(selector);
-
+const $ = selector => document.querySelector(selector);
+const create = element => document.createElement(element);
+const docLoc = "https://raw.githubusercontent.com/codidact/docs/master/User-Help/CodidactMainPageFAQ.md";
+const svgButtonArrow = `<svg alt="Arrow down" mlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512" class="button-expand"><path fill="currentColor" d="M413.1 222.5l22.2 22.2c9.4 9.4 9.4 24.6 0 33.9L241 473c-9.4 9.4-24.6 9.4-33.9 0L12.7 278.6c-9.4-9.4-9.4-24.6 0-33.9l22.2-22.2c9.5-9.5 25-9.3 34.3.4L184 343.4V56c0-13.3 10.7-24 24-24h32c13.3 0 24 10.7 24 24v287.4l114.8-120.5c9.3-9.8 24.8-10 34.3-.4z"></path></svg>`;
```

Comment:
this is a bit gross. You should add this as HTML and grab the element if you need to manipulate it

Reply:
Not sure what that would solve, why not have it as a const string and then use it? I'll make this an SVG asset which is better I think

Reply:
because you are inserting an image and causing reflow with javascript instead of having it load on page load and not cause reflow. In general, you should load your images in one way unless there is a specific reason.